heron/uploaders/src/java/com/twitter/heron/uploader/docker/sample.yaml

```diff
@@ -0,0 +1,11 @@
+# uploader class for transferring the topology jar/tar files to storage
+heron.class.uploader:         com.twitter.heron.uploader.docker.DockerUploader
+
+#base image
+heron.uploader.docker.base:                      ubuntu:latest
```

Comment:
@ajorgensen when you do go down that route you'll find that the heron executor tries to run chmod +x on a number of the base files. That won't work if they're not owned by the user who's running the executor. That's definitely not going to be the case for a Docker Image deployed to Aurora. It's a simple change to the executor to treat those as non-terminal failures.
I can put together a Docker file for some base images easily enough, however that's not going to change this Uploader at all. I'm working on figuring out why it's not working for you but the problem I'm currently having is that I can't get the docker build to complete locally, it just hangs with 177 tasks remaining.